ebu_tt_live/node/delay.py

```diff
@@ -2,8 +2,9 @@
 import six
 import logging
 from .base import AbstractCombinedNode
-from datetime import timedelta
+from datetime import timedelta, datetime
```

Comment:
Not unless you tell it to - this is slightly confusing because the thing being imported and the module have the same name in the case of `datetime`.